packages/runtime/src/utils/load.ts

```diff
@@ -20,9 +20,9 @@ export async function loadEsmEntry({
       if (!remoteEntryExports) {
         // eslint-disable-next-line no-eval
         new Function(
-          'resolve',
+          'resolve', 'reject',
```

Comment:
can you confirm that rejecting works when passing it a bad entry? I tried this last time and it seemed unhappy with multiple args passes like this.

Reply:
```js
console.log(
    await (
        new Promise(
            (resolve, reject) => {
                new Function(
                    'resolve', 'reject',
                    `import("https://esm.sh/react@18.5.0").then(
                        (res) => {
                            console.log('Success');
                            resolve(res);
                        },
                        (error) => {
                            console.log('Inside error');
                            reject(error);
                        }
                    )`
                )(resolve, reject); 
            }
        )
    )
);
```
The following script works! Let me try to add a test for it.